lib/shelley/src/Cardano/Wallet/Shelley/Launch.hs

```diff
@@ -452,49 +549,76 @@ withBFTNode
     -> FilePath
     -- ^ Parent state directory. Node data will be created in a subdirectory of
     -- this.
+    -> Int
+    -- ^ Which node (assumes key files are indexed by this number)
     -> NodeParams
     -- ^ Parameters used to generate config files.
     -> (FilePath -> Block -> (NetworkParameters, NodeVersionData) -> IO a)
     -- ^ Callback function with genesis parameters
     -> IO a
-withBFTNode tr baseDir (NodeParams severity systemStart (port, peers)) action =
+withBFTNode
+    tr
+    baseDir
+    i
```

Comment:
I don't understand why this index is added. This suggests there could be several BFT nodes, but it it's not the case.

Reply:
Yes, not needed anymore 👍